File: python/check_connection.py
import requests
import os
base=os.environ['GRIDDB_WEBAPI_URL']
creds=os.environ['USER_PASS']

url = base+"/checkConnection"

payload = {}
headers = {
  'Authorization': 'Basic '+creds,
  'User-Agent': 'PostmanRuntime/7.29.0'
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.status_code)

File: nodejs/checkConnection.js
require('dotenv').config()
var request = require('request');
const base = process.env.GRIDDB_WEBAPI_URL
const creds = process.env.USER_PASS
const options = {
  'method': 'GET',
  'url': base+'/checkConnection',
  'headers': {
    'Authorization': 'Basic '+creds,
    'User-Agent':'PostmanRuntime/7.29.0'
  }
};
request(options, function (error, response) {
  if (error) throw new Error(error);
  console.log("Response Status Code: ", response.statusCode);
});